tests/static/phpstan.neon

```diff
@@ -0,0 +1,6 @@
+parameters:
+	excludes_analyse:
```

Comment:
It may be easier/more consistent to add:
```yaml
  level: 5
  paths:
    - src
```
This way there's less to worry about passing on the command line by default

Reply:
Make sense, updated. Thank you.

Reply:
This only works for the newer versions of PHPstan and doesn't work for versions compatible with PHP 7.0. So I didn't add level and paths to `phpstan7.0.neon` file.